nestedtensor/csrc/functions.cpp

```diff
@@ -194,8 +194,8 @@ NestedTensor cross_entropy(NestedTensor input,
 }
 
 NestedTensor interpolate(NestedTensor input,
-                         c10::optional<std::vector<int64_t>> size,
-                         c10::optional<std::vector<double>> scale_factor,
+                         c10::optional<at::IntArrayRef> size,
```

Comment:
Does this align with the aten definition?

Reply:
[yes](https://github.com/pytorch/pytorch/blob/master/torch/csrc/api/include/torch/nn/options/upsampling.h#L67)